Sources/SyrupCore/Generator/Renderer.swift

```diff
@@ -68,27 +68,25 @@ open class Renderer {
 		}
 	}
 	
-	func renderQueries(intermediateRepresentation: IntermediateRepresentation, selectionSets: SelectionSetVisitor.Results) throws -> [String] {
+	func renderOperations(intermediateRepresentation: IntermediateRepresentation, selectionSets: SelectionSetVisitor.Results, operations: [IntermediateRepresentation.OperationDefinition]) throws -> [String] {
 		var rendered: [String] = []
-		for query in intermediateRepresentation.operations.queries {
-			rendered.append(try renderQuery(query: query, intermediateRepresentation: intermediateRepresentation, querySelections: selectionSets.operation(named: query.name)))
+		// just flat map these operations
+		for query in operations {
+			rendered.append(try renderOperation(operation: query, intermediateRepresentation: intermediateRepresentation, querySelections: selectionSets.operation(named: query.name)))
 		}
```

Comment:
Maybe call this `for operation in operations` (or `for op in operations`) since you're renaming queries to operations in other places.